### PR TITLE
Clarify we also support s3 as log storage

### PR DESCRIPTION
**Pull request by alvaroaleman**

### CODE REVIEW

File: prow/cmd/plank/README.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ plank:
         entrypoint: gcr.io/k8s-prow/entrypoint:v20190221-d14461a
         sidecar: gcr.io/k8s-prow/sidecar:v20190221-d14461a
       gcs_configuration: # configuration for uploading job results to GCS
-        bucket: <bucket-name>
+        bucket: <bucket-name> or s3://<bucket-name>
         path_strategy: explicit # or `legacy`, `single`
         default_org: <github-org> # should not need this if `strategy` is set to explicit
         default_repo: <github-repo> # should not need this if `strategy` is set to explicit
-      gcs_credentials_secret: <secret-name> # the name of the secret that stores the GCP service account credential JSON file, it expects the secret's key to be `service-account.json`
+      gcs_credentials_secret: <secret-name> # the name of the secret that stores cloud provider crdentials
```
Comment (petr-muller) [inline prow/cmd/plank/README.md:37]:
```suggestion
      gcs_credentials_secret: <secret-name> # the name of the secret that stores cloud provider credentials
```

Comment (alvaroaleman) [reply prow/cmd/plank/README.md]: fixed

**Review by petr-muller — APPROVED** [review]

**Pull request merged**